app/views/studies/_initialize_primary_data_form.html.erb

```diff
@@ -108,24 +109,23 @@
           if (selectedTaxon !== '') {
               $.getJSON('<%= get_taxon_path %>?taxon=' + selectedTaxon, function(taxon) {
                   var humanDataSelect = $('#primary_data_form_<%= study_file.id %>').find('.human-data');
-                  if ( restrictedTaxons.includes(taxon.taxon_identifier) ) {
+                  if ( taxon.restricted ) {
```

Comment:
Nit: `if ( taxon.restricted )` should be `if (taxon.restricted)`.

Reply:
I'll fix that.